doc/api/date/date-formatter.md

```diff
@@ -20,9 +20,9 @@ A JSON object including one of the following.
 > Skeleton provides a more flexible formatting mechanism than the predefined
 > list `full`, `long`, `medium`, or `short` represented by date, time, or
 > datetime.  Instead, they are an open-ended list of patterns containing
-> only [date
-> field](http://www.unicode.org/reports/tr35/tr35-dates.html#Date_Field_Symbol_Table)
-> information, and in a canonical order. For example:
+> only date field information, and in a canonical order. For a complete list of skeleton patterns [check the unicode CLDR documentation](http://www.unicode.org/reports/tr35/tr35-dates.html#Date_Field_Symbol_Table).
+> 
```

Comment:
Please, can you break lines at 80 char. We don't have a explicit style rule for that, but I suggest we keep it consistent with the above lines.